contracts/mocks/core/lib/ConstantPriceAuctionLibrary.sol

```diff
@@ -0,0 +1,64 @@
+/*
+    Copyright 2018 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.4.24;
+
+
+/**
+ * @title LinearAuctionLibrary
+ * @author Set Protocol
+ *
+ * Library used in rebalancing auctions to calculate price based off of a linear curve
+ *
+ */
+
+contract ConstantPriceAuctionLibrary {
```

Comment:
If a manager proposes a constant price library, couldn't it be very likely that a rebalance could never end?

Reply:
It's for testing, not for actual deployment. Why I threw it in the mocks folder.